app/controllers/api/v0/swagger/models/search.rb

```diff
@@ -76,29 +76,37 @@ module Api::V0::Swagger::Models::Search
     property :overall_took do
       key :type, :integer
       key :readOnly, true
-      key :description, "How long the request took inside Open-Search, including ES 'took' (ms)"
+      key :description, "How long the request took inside OpenSearch, including 'took' (ms)"
     end
     property :took do
       key :type, :integer
       key :readOnly, true
-      key :description, "How long the request took inside Elasticsearch (ms)"
+      key :description, "How long the request took inside OpenSearch (ms)"
     end
     property :timed_out do
       key :type, :boolean
       key :readOnly, true
-      key :description, "Whether the request in Elasticsearch timed out"
+      key :description, "Whether the request in OpenSearch timed out"
     end
     property :_shards do
       key :type, :object
       key :readOnly, true
-      key :description, "Shard stats from Elasticsearch"
+      key :description, "Shard stats from OpenSearch"
     end
     property :hits do
       key :required, [:total, :hits]
       property :total do
-        key :type, :integer
-        key :readOnly, true
-        key :description, "The number of hits"
+        key :required, [:value, :relation]
+        property :value do
+          key :type, :integer
+          key :readOnly, true
+          key :description, "The exact or approximate of hits"
+        end
+        property :relation do
+          key :type, :string
+          key :readOnly, true
+          key :description, "Modifier for value"
```

Comment:
are we changing the schema of the response here? this'll require a new client and rex updates to be compatible right?

Reply:
I added a commit to revert this change and added a translation in the controller so we can keep using v0